### DISCUSSION TITLE
BrowserFS Deprecated

### DISCUSSION THREAD

**Discussion by james-pre** [DustinBrett/daedalOS#428] [Ideas]
Hi there, I'm the maintainer of BrowserFS. I've been going through the dependents of browserfs.

Please see https://github.com/browser-fs/NOTICE

Would you be interested in updating your project to use the updated fork, [ZenFS](https://github.com/zen-fs/core)?

**Comment by DustinBrett** [comment]
Thanks for the suggestion @james-pre. I will look into ZenFS for future changes to my code. The version I use is already heavily modified and in my package.json I refer to an older version almost exclusively for typing, and for the `Stats` class. I was keeping up with BrowserFS for quite a while but early on when you took over it began to deviate too much to maintain so I forked my own.

**Comment by james-pre** [comment]


**Reply by DustinBrett** [reply]
> [@DustinBrett](https://github.com/DustinBrett) I've updated my GitHub username from [@dr-vortex](https://github.com/dr-vortex) to [@james-pre](https://github.com/james-pre). Please update your comment(s), and I can delete this one. I'm writing this comment when I can't make the change myself.

The academy revoked your PhD? :-)

Sure I will update this comment thread at least.
